static/js/map.js

```diff
@@ -387,6 +391,17 @@ function getTypeSpan(type) {
 
 function openMapDirections(lat, lng) { // eslint-disable-line no-unused-vars
     var url = 'https://www.google.com/maps/dir/?api=1&destination=' + lat + ',' + lng
+    switch (directionProvider) {
+        case 'apple':
+            url = 'http://maps.apple.com/?ll=' + lat + ',' + lng
```

Comment:
I haven't looked into the other urls yet but shouldn't this be: `'http://maps.apple.com/?daddr=' + lat + ',' + lng`